bindings/wasm/examples-account/src/revoke_vc.ts

```diff
@@ -16,9 +16,9 @@ import {
 
```

Comment:
> Decide what to do with the validate_university_degree tutorial.

No idea. It has quite a lot of work put into it, so it would be a real shame to lose it.

Reply:
Let's reach out and offer to maintain a fork of the repository under our control.

Reply:
I don't think the tutorial should be a blocker for the release.

Reply:
Adrian gave us permission to take over the repository. Let's tackle this sperately.

Reply:
tracked in #907